packages/cache/src/useSuspenseCache.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo, useSyncExternalStore } from 'react'
+import type { CacheKey, CacheOptions, ResolvedData } from './types'
+import { useCache } from './useCache'
+
+/**
+ * @experimental This is experimental feature.
+ */
+export const useSuspenseCache = <TData, TCacheKey extends CacheKey>(
+  options: CacheOptions<TData, TCacheKey>
+): ResolvedData<TData> => {
```

Comment:
```suggestion
): UseSuspenseCacheResult<TData> => {
```

Reply:
great name!!

Reply:
However, this type is used in both useSuspenseCache and SuspenseCache, so limiting it to just UseSuspenseCache may not be appropriate. How about SuspenseCacheResult?

Reply:
Cool 👍